.changeset/seven-hotels-collect.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+Fixes a problem with custom sounds causing files to not be playable when using filesystem storage.
```

Comment:
Based on the structure guidelines it should be:

> Fixes a problem with custom sounds (What) that prevents files from being playable when using filesystem storage (Why) by adjusting the file access method (How)

Reply:
Actually, i don't mind changing. But since the original CS has the 2 of 3 elements (what why) like all the other CSs out there, we should be good.

Thanks!